include/ibamr/IBImplicitStaggeredHierarchyIntegrator.h

```diff
@@ -150,11 +156,113 @@ class IBImplicitStaggeredHierarchyIntegrator : public IBHierarchyIntegrator
 
 private:
     /*!
-     * \brief Default constructor.
-     *
-     * \note This constructor is not implemented and should not be used.
-     */
-    IBImplicitStaggeredHierarchyIntegrator();
+	 * \brief A StaggeredStokesSolver that does nothing.
```

Comment:
> On Apr 24, 2015, at 12:28 AM, Amneet Bhalla <notifications@github.com> wrote:
> 
> In include/ibamr/IBImplicitStaggeredHierarchyIntegrator.h <https://github.com/IBAMR/IBAMR/pull/6#discussion_r29023070>:
> 
> > @@ -150,11 +156,113 @@ class IBImplicitStaggeredHierarchyIntegrator : public IBHierarchyIntegrator
> >  
> >  private:
> >      /*!
> > -     * \brief Default constructor.
> > -     *
> > -     * \note This constructor is not implemented and should not be used.
> > -     */
> > -    IBImplicitStaggeredHierarchyIntegrator();
> > +	 * \brief A StaggeredStokesSolver that does nothing.
> We need to get info like u_problem_coef, nullspace info etc from INSHierarchyIntegrator, but we do not need to create any StokesSolver within ins hierarchy integrator. I don't think we need to change design, because to me provision for registering external stokes solver with ins integrator seems to be precisely for these situations...
> 

You can also reset the Stokes operator and PC with a standard Stokes Krylov solver. Does initialization not work correctly for the IB part?